doc/whats_new/v1.1.rst

```diff
@@ -305,6 +305,10 @@ Changelog
   `__init__` but in `.fit()`.
   :pr:`21888` by :user:`iofall <iofall>` and :user: `Arisa Y. <arisayosh>`.
```

Comment:
```suggestion
  :pr:`21888` by :user:`iofall <iofall>` and :user:`Arisa Y. <arisayosh>`.
```

Reply:
Thank you for the feedback I have made these changes!